llvm/lib/CodeGen/GlobalISel/CallLowering.cpp

```diff
@@ -751,6 +751,8 @@ bool CallLowering::handleAssignments(ValueHandler &Handler,
     const LLT NewLLT = Handler.isIncomingArgumentHandler() ? LocTy : ValTy;
     const EVT OrigVT = EVT::getEVT(Args[i].Ty);
     const LLT OrigTy = getLLTForType(*Args[i].Ty, DL);
+    const LLT PointerTy = LLT::pointer(MF.getDataLayout().getAllocaAddrSpace(),
```

Comment:
```
unsigned AllocaAS = MF.getDataLayout().getAllocaAddrSpace();
const LLT PointerTy = LLT::pointer(AllocaAS, DL.getPointerSizeInBits(AllocaAS));
```

Reply:
I have done this with a different name. Can the name stay or should I rename it to the name specified by you?

Reply:
No worries. It is fine.